kernel/src/main/java/org/kframework/parser/inner/kernel/KSyntax2Bison.java

```diff
@@ -117,10 +117,8 @@ public static Module transformByPriorityAndAssociativity(Module module) {
   }
 
   public static void writeParser(Module module, Module disambModule, Scanner scanner, Sort start, File path, boolean glr, long stackDepth, KExceptionManager kem) {
-    if (!glr && module.att().contains("not-lr1")) {
-        kem.registerInnerParserWarning(ExceptionType.NON_LR_GRAMMAR, "Importing a module that is tagged as not being LR(1) when using Bison's LR(1) parser generator: " + module.att().get("not-lr1"));
-        // print so it appears before we call bison which may not terminate
-        kem.print();
+    if (module.att().contains("not-lr1")) {
+        kem.registerInnerParserWarning(ExceptionType.NON_LR_GRAMMAR, "Importing a module that is tagged as not being LR(1) when using Bison's parser generator: " + module.att().get("not-lr1") + " Module will not be included in Bison grammar.");
```

Comment:
`kem.registerBisonParserWarning(ExceptionType.NON_LR_GRAMMAR, "Skipping module X tagged as not-lr1")`
The error message is way too wordy. It should give you some information on what happened. The full explanation should be somewhere in the documentation (maybe pending-documentation.md)